content/en/brupop/1.3.x/setup/configure/index.markdown

```diff
@@ -0,0 +1,280 @@
+Helm reduces the configuration burden for Brupop substantially with few down sides, so this documentation focuses on configuration with helm.
+If you choose to not use helm, refer to the {{< github-link-at-version url="https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/vx.x.x/bottlerocket-update-operator.yaml" project="brupop" >}}pre-baked manifest for an example{{< /github-link-at-version >}}.
+
+## Required Configuration
+
+### Label nodes
+
+{{% alert title="Warning" color="warning" %}}
+You can fully install Brupop but if you do not apply the proper node labels the operator will not update your nodes.
+{{% /alert %}}
+
+[Kubernetes node labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) control which nodes Brupop updates; specifically, the label `bottlerocket.aws/updater-interface-version=2.0.0` dictates which nodes in the cluster get automatic updates.
+
+You can label nodes using {{< cross-project-current-link url="/en/os/x.x.x/api/settings/kubernetes/#node-labels" >}}`settings.kubernetes.node-labels`{{</ cross-project-current-link>}} with TOML {{< cross-project-current-link url="/en/os/x.x.x/concepts/api-driven/#user-data" >}}(including instance user data){{</ cross-project-current-link>}}, using `apiclient` in a host container, or `kubectl`.
+
+#### Label a node with `apiclient`
+
+```shell
+apiclient set settings.kubernetes.node-labels.bottlerocket.aws/updater-interface-version=2.0.0
+```
+
+#### Label all nodes when starting an EKS cluster with `eksctl`
+
+```yaml
+...
+nodeGroups:
+  - name: name-of-your-nodegroup
+    labels: { bottlerocket.aws/updater-interface-version: 2.0.0 }
+...
+```
+
+#### Labeling nodes with `kubectl`
+
+#### Label a single node
+
+```shell
+# replace MY_NODE_NAME with the name of your node
+kubectl label node MY_NODE_NAME bottlerocket.aws/updater-interface-version=2.0.0
+```
+
+##### Label all nodes
+
+If you are running Bottlerocket on all nodes in your cluster, you can use `kubectl` to label all nodes at once:
+
+```shell
+kubectl label node $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}') bottlerocket.aws/updater-interface-version=2.0.0
+```
+
+#### Labeling a node with the Bottlerocket API
+
+Add the following TOML to your instance user data:
+
+```TOML
+...
+[settings.kubernetes.node-labels]
+"bottlerocket.aws/updater-interface-version" = 2.0.0
+...
+```
+
+From the control container, run the following:
+
+```shell
+apiclient set settings.kubernetes.node-labels.bottlerocket.aws/updater-interface-version=2.0.0
+```
+
+## Optional Configuration
+
+### API Server Ports
+
+__Helm Configuration__: `apiserver_internal_port` for internal traffic, `apiserver_service_port` for node agent traffic.
+
+By default, the operator’s API server uses port `8443` for internal traffic and port `443` for node agents, but you can change these ports via this configuration. Both ports must be set or the operator will fail to start.
```

Comment:
Providing a reason on why this port is needed will be helpful, since there could be users running brupop on highly confined environments.